### PR TITLE
Added May with httparse

### PR DESCRIPTION
**Pull request by gkbrk**

### CODE REVIEW

File: frameworks/Rust/may-httparse/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        path.clear();
+        path.push_str(req.path.unwrap_or("/"));
+        return Some(i);
+    }
+
+    None
+}
+
+fn main() {
+    may::config().set_io_workers(4);
```
Comment (nayato) [inline frameworks/Rust/may-httparse/src/main.rs]: use number of cpus here, e.g. https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Rust/hyper/src/main.rs#L65

File: frameworks/Rust/may-httparse/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                        _ => "Cannot find page"
+                    };
+
+                    let s = format!("\
+                        HTTP/1.1 200 OK\r\n\
+                        Server: May\r\n\
+                        Date: {}\r\n\
+                        Content-Length: {}\r\n\
+                        \r\n\
+                        {}", time::now().rfc822(), response.len(), response);
```
Comment (nayato) [inline frameworks/Rust/may-httparse/src/main.rs]: switch to httpdate here (for context: https://github.com/swindon-rs/tk-http/issues/32)

**Comment by fafhrd91** [comment]
I wonder, is this framework benchmarks?

**Comment by nbrady-techempower** [comment]
Feel free to re-open if you plan on fixing

**Pull request closed without merging**